src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml

```diff
@@ -0,0 +1,88 @@
+                </GeometryDrawing>
+              </DrawingImage>
+            </Image>
+          </ToggleButton>
+          <ToggleButton Classes="filter-text-box-toggle" IsChecked="{Binding $parent[controls:FilterTextBox].UseRegexFilter}">
+            <Image>
+              <DrawingImage>
+                <GeometryDrawing Brush="{Binding $parent[ToggleButton].Foreground}">
+                  M10.0122 2H10.9879V5.11346L13.5489 3.55609L14.034 4.44095L11.4702 6L14.034 7.55905L13.5489 8.44391L10.9879 6.88654V10H10.0122V6.88654L7.45114 8.44391L6.96606 7.55905L9.5299 6L6.96606 4.44095L7.45114 3.55609L10.0122 5.11346V2ZM2 10H6V14H2V10Z
+                </GeometryDrawing>
+              </DrawingImage>
+            </Image>
+          </ToggleButton>
+        </StackPanel>
+      </Template>
+    </Setter>
+  </Style>
+
+  <Style Selector="TextBox.filter-text-box Button.textBoxClearButton">
+    <Setter Property="IsVisible" Value="False" />
+  </Style>
+  <Style Selector="TextBox.filter-text-box[AcceptsReturn=False][IsReadOnly=False]:focus:not(TextBox:empty) Button.textBoxClearButton">
+    <Setter Property="IsVisible" Value="True" />
+  </Style>
+
+  <Style Selector="ToggleButton.filter-text-box-toggle > Image">
+    <Setter Property="Width" Value="16" />
+    <Setter Property="Height" Value="16" />
+    <Setter Property="Opacity" Value="0.5" />
+  </Style>
+  <Style Selector="ToggleButton.filter-text-box-toggle:pointerover > Image">
+    <Setter Property="Opacity" Value="0.7" />
+  </Style>
+  <Style Selector="ToggleButton.filter-text-box-toggle:pressed > Image, ToggleButton.filter-text-box-toggle:checked > Image">
+    <Setter Property="Opacity" Value="0.7" />
+  </Style>
+
+  <Style Selector="ToggleButton.filter-text-box-toggle">
+    <Setter Property="Padding" Value="0,1" />
```

Comment:
I think it would be nice to set `Cursor` to `Arrow` here. Since right now cursor from `TextBox` kicks in.

Reply:
Actually VS Code is using a hand cursor in this case.

Reply:
Set it to Hand cursor